frontend-react/src/hooks/UseCreateFetch.ts

```diff
@@ -60,17 +62,18 @@ export const auxExports = {
     createTypeWrapperForAuthorizedFetch,
 };
 
-export const useCreateFetch = (
-    oktaToken: Partial<AccessToken>,
-    activeMembership: MembershipSettings,
-): AuthorizedFetchTypeWrapper => {
+export const useCreateFetch = (): AuthorizedFetchTypeWrapper => {
+    const { activeMembership, authState = {} } = useSessionContext();
+    const { fetchHeaders } = useAppInsightsContext();
+
     const generator = useCallback(
         () =>
             auxExports.createTypeWrapperForAuthorizedFetch(
-                oktaToken as Partial<AccessToken>,
+                authState?.accessToken as Partial<AccessToken>,
                 activeMembership as MembershipSettings,
+                fetchHeaders,
             ),
-        [oktaToken, activeMembership],
+        [authState?.accessToken, activeMembership, fetchHeaders],
     );
 
     return generator;
```

Comment:
@jpandersen87 I logged in fine, but then once I went to `/admin/settings` I get:

![Screenshot 2023-10-20 at 3 00 14 PM](https://github.com/CDCgov/prime-reportstream/assets/516877/07dc7852-af4b-4247-99ca-f72f299b0b0a)

Reply:
For me, I get the following:
![Screenshot 2023-10-20 at 3 05 13 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/c889ca4b-413f-4e0d-93f5-e31009287f3e)